### ISSUE TITLE
Copyright attribution

### ISSUE THREAD

**Issue by AaronBruce** [indexeddbshim/IndexedDBShim#293]
Hi,
What is the correct copyright attribution (party and year(s))?
I don't see a copyright statement in either license file (Apache or MIT).
Thanks,
Aaron

**Comment by brettz9** [comment]
Thanks for the question.

I've added `Copyright 2014-2017 Parashuram N., Brett Zamir` to both files along with adding the appropriate Apache header to `master`.  (The copyright files were added in 2014.)

If you want to add a last name, @axemclion instead of "Parashuram N.", let me know what it is. :)

Previous contributors can add their names here and into the `package.json` `contributors` field if desired (noting that adding one's name to copyright is not necessary to be able to use the already lenient Apache or MIT licenses, merely to declare that, as a copyright holder, one has the right to relicense one's contributions under other license terms).

I'll close for now though feel free to add further comments.

**Comment by brettz9** [comment]
Thanks for the question.

I've added `Copyright 2014-2017 Parashuram N., Brett Zamir` and the appropriate Apache header to `master`.  (The copyright files were added in 2014.)

If you want to add a last name, @axemclion instead of "Parashuram N.", let me know what it is. :)

Previous contributors can add their names here and into the `package.json` `contributors` field if desired (noting that adding one's name to copyright is not necessary to be able to use the already lenient Apache or MIT licenses, merely to declare that, as a copyright holder, one has the right to relicense one's contributions under other license terms).

I'll close for now though feel free to add further comments.
